test/plugins/features/movies/controller.test.js

```diff
@@ -39,4 +48,71 @@ describe('movie controller', () => {
 
   });
 
+  describe('list', () => {
+
+    it('should properly sort based on query params', () => {
```

Comment:
minor minor minor thing but we omit the `should` from in front of `it` clause descriptions since they all `should` do something.